tests/test_queue.py

```diff
@@ -306,6 +308,20 @@ def test_dequeue_any_ignores_nonexisting_jobs(self):
         )
         self.assertEqual(q.count, 0)
 
+    def test_redis_mode(self):
+        q = Queue()
+        ret = q.redis_mode(self.testconn)
+        logger.info('wqlu redis mode is {}'.format(ret))
+        self.assertIsNotNone(ret)
+        self.assertEqual(ret, 'standalone')
+
+    def test_is_redis_cluster(self):
+        q = Queue()
+        ret = q.is_redis_cluster(self.testconn)
+        logger.info('wqlu is redis cluster {}'.format(ret))
+        self.assertIsNotNone(ret)
+        self.assertEqual(ret, False)
```

Comment:
Is there a way we can test this with Redis cluster? Can we perhaps fake a Redis cluster connection?

Reply:
I will research and try.